q2_feature_classifier/_blast.py

```diff
@@ -0,0 +1,77 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2017, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import tempfile
+import pandas as pd
+from os.path import join
+from q2_types.feature_data import (
+    FeatureData, Taxonomy, Sequence, DNAFASTAFormat)
+from qiime2.plugin import Int, Str, Float
+from .plugin_setup import plugin
+from ._consensus_assignment import (_consensus_assignments,
+                                    _parse_params,
+                                    _run_command)
+
+
+def blast(query: DNAFASTAFormat, reference_reads: DNAFASTAFormat,
+          reference_taxonomy: pd.Series, maxaccepts: int=10, min_id: int=0.8,
+          strand: str='both', evalue: float=0.001, t_delim: str=';',
+          min_consensus: float=0.51, unassignable_label: str="Unassigned",
+          num_threads: str=1, params: str=None) -> pd.DataFrame:
+
+    min_id = min_id * 100
```

Comment:
You should raise an error here if `min_id` is outside the range of [0.0, 1.0]. A common mistake is that users will pass a percentage (e.g., 97) rather than a fractional value (e.g., 0.97). While this check isn't perfect, it'd catch most instances. You should also include examples of how to use it in the help text when you register the method, to help users avoid this mistake.